src/main/webapp/myprofile.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Student Recommendations</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="/scripts/myprofile.js"></script>
+    <script
+      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDME967qxzDl6dYJTT_d25n6yPHM1LOehw&callback=initMap&libraries=&v=weekly"
+      defer
+    ></script>
+    <link rel="stylesheet" href="/styles/fonts.css">
+    <link rel="stylesheet" href="/styles/common.css">
+    <link rel="stylesheet" href="/styles/myprofile.css">
+  </head>
+  <body onload="initMap()">
+    <div class="header">
+      <h1>Student Recs</h1>
+      <h3>a recommendation system for students, by students!</h3>
+      <!-- Navigation Bar -->
+      <nav>
+        <ul>
+          <li><a href="/recmap.html">Recommendation Map</a></li>
+          <li><a href="/userdir.html">User Directory</a></li>
+          <li><a class="active" href="/myprofile.html">MyProfile</a></li>
+        </ul>
+      </nav>
+    </div>
+    <h1>User Directory</h1>
+    <div class="content">
+      <div class="wrapper">
+        <!-- User Details -->
+        <div class="details-container">
+            <img src="/images/avatar.jpg"></img>
+            <h2>Name</h2>
+            <h3>Department</h3>
+            <p>Hello I am interested in __ please feel free to contact me for __</p>
```

Comment:
I'm not sure what this bit is for. Maybe add a TODO if there's some future work needed?

Reply:
This is for bio :) the one I put is just a bio example